parlai/messenger/core/message_sender.py

```diff
@@ -164,10 +166,10 @@ def send_sender_action(self, receiver_id, action):
     def send_read(self, receiver_id):
         self.send_sender_action(receiver_id, "mark_seen")
 
-    def typing_on(self, receiver_id):
-        self.send_sender_action(receiver_id, "typing_on")
+    def typing_on(self, receiver_id, persona_id=None):
+        self.send_sender_action(receiver_id, "typing_on", persona_id)
 
-    def send_fb_payload(self, receiver_id, payload, quick_replies=None):
+    def send_fb_payload(self, receiver_id, payload, quick_replies=None, persona_id=None):
```

Comment:
Nit: line too long, travis build is failing lint due to this.

Reply:
Fixed!